src/constants.ts

```diff
@@ -1,5 +1,6 @@
 export const DR_API = '/api/digital-roadmap/v1';
 export const DR_RELEASE_NOTES = '/release-notes';
+export const DR_LIFECYCLE = '/lifecycle/systems';
```

Comment:
```suggestion
export const DR_LIFECYCLE = '/lifecycle/systems/';
```
The api endopoint [has changed](https://github.com/RedHatInsights/digital-roadmap-backend/commit/e3fc115087714df42adc4a958fc387aa702a9c2a#diff-94f4d76d854beffce11d71669916f876ba7467a6a6a50abb6d13af2c09b20e3aL8) and we were getting redirect - bad data in `lifecycleData` which caused `lifecycleData.map is not a function`